ansible/roles/testbed/tasks/main.yml

```diff
@@ -52,16 +52,30 @@
     - Private
     state: disabled
 
-- name: stop docker
+- name: Stop Docker
   win_service:
-      name: docker
-      state: stopped
+    name: docker
+    state: stopped
 
-- name: remove container networks
+- name: Remove container networks
   win_shell: |
-        Get-NetNat | Remove-NetNat -Confirm:$false
-        Get-ContainerNetwork | Remove-ContainerNetwork -Force
+    Get-NetNat | Remove-NetNat -Confirm:$false
+    Get-ContainerNetwork | Remove-ContainerNetwork -Force
 
-- name: enable testsigning
-  win_shell: >
-      Bcdedit.exe -set TESTSIGNING ON
+- name: Enable testsigning
+  win_shell: |
+    Bcdedit.exe -set TESTSIGNING ON
+
+- name: Copy IIS TCP test Dockerfile
+  win_copy:
+    src: "{{dockerfile_dir}}"
+    dest: C:\
+
+# docker_image module doesn't work on Windows
+- name: Build IIS TCP test Docker image
+  register: output
+  win_shell: |
+    docker build -t iis-tcptest "C:\\{{dockerfile_dir}}"
```

Comment:
How does the `docker build` work after stopping docker service? I hope it doesn't auto-start here, because we need docker to be disabled when creating a template.